sdk/core/azure-core/test/ut/base64_test.cpp

```diff
@@ -125,3 +125,21 @@ TEST(Base64, Roundtrip)
     EXPECT_EQ(Convert::Base64Decode(Convert::Base64Encode(data)), data);
   }
 }
+
+TEST(Base64, InvalidDecode)
+{
+  EXPECT_NO_THROW(Convert::Base64Decode(Convert::Base64Encode(std::vector<uint8_t>{})));
+  EXPECT_NO_THROW(Convert::Base64Decode(""));
+  EXPECT_THROW(Convert::Base64Decode("@#!%"), std::runtime_error);
+  EXPECT_THROW(Convert::Base64Decode("ABCD%GA="), std::runtime_error); // cspell:disable-line
```

Comment:
nit.
You can add spell disable before this line and enable after line 136 to avoid the line by line disable.

Reply:
I was trying to decide if a blanket exception was better than a line-by-line exception... I'll make it blanket.